rust/processor/src/grpc_stream.rs

```diff
@@ -251,8 +288,11 @@ pub async fn create_fetcher_loop(
         "[Parser] Successfully connected to GRPC stream",
     );
 
-    let mut last_fetched_version = batch_start_version as i64 - 1;
-    let mut batch_start_version = batch_start_version;
+    let mut last_fetched_version = starting_version as i64 - 1;
```

Comment:
We have both `next_version_to_fetch` and `last_fetched_version`, could we get rid of one of them